dev-haskell/exceptions/exceptions-0.10.10.ebuild

```
@@ -0,0 +1,31 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.5.1.9999
+
+CABAL_HACKAGE_REVISION=1
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
```

Remove `test-suite` if you're commenting out its dependencies.

```suggestion
CABAL_FEATURES="lib profile haddock hoogle hscolour"
```